common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -370,6 +371,83 @@ class RssConf(loadDefaults: Boolean) extends Cloneable with Logging with Seriali
+  def flushAvgTimeMinimumCount: Int = get(WORKER_FLUSHER_AVGFLUSHTIME_SLIDINGWINDOW_MINCOUNT)
+  def diskMinimumReserveSize: Long = get(WORKER_DISK_RESERVE_SIZE)
+  def deviceMonitorEnabled: Boolean = get(WORKER_DEVICE_MONITOR_ENABLED)
+  def deviceMonitorCheckList: Seq[String] = get(WORKER_DEVICE_MONITOR_CHECKLIST)
+  def diskCheckIntervalMs: Long = get(WORKER_DISK_CHECK_INTERVAL)
+  def sysBlockDir: String = get(WORKER_DEVICEMONITOR_SYS_BLOCKDIR)
+  def createWriterMaxAttempts: Int = get(WORKER_WRITER_CREATE_MAXATTEMPTS)
+  def workerBaseDirPrefix: String = get(WORKER_BASE_DIR_PREFIX)
+  def workerBaseDirNumber: Int = get(WORKER_BASE_DIR_COUNT)
+
+  /**
+   * @return workingDir, usable space, flusher thread count, disk type
+   *         check more details at CONFIGURATION_GUIDE.md
+   */
+  def workerBaseDirs: Seq[(String, Long, Int, Type)] = {
+    // I assume there is no disk is bigger than 1 PB in recent days.
+    val defaultMaxCapacity = Utils.byteStringAsBytes("1PB")
+    get(WORKER_STORAGE_DIRS).map { storageDirs: Seq[String] =>
+      storageDirs.map { str =>
+        var maxCapacity = defaultMaxCapacity
+        var diskType = HDD
+        var flushThread = -1
+        val (dir, attributes) = str.split(":").toList match {
+          case _dir :: tail => (_dir, tail)
+          case nil => throw new IllegalArgumentException(s"Illegal storage dir: $nil")
+        }
+        attributes.foreach {
+          case capacityStr if capacityStr.toLowerCase.startsWith("capacity=") =>
+            maxCapacity = Utils.byteStringAsBytes(capacityStr.split("=")(1))
+          case diskTypeStr if diskTypeStr.toLowerCase.startsWith("disktype=") =>
+            diskType = Type.valueOf(diskTypeStr.split("=")(1))
+            if (diskType == Type.MEMORY) {
+              throw new IOException(s"Invalid diskType: $diskType")
+            }
+          case threadCountStr if threadCountStr.toLowerCase.startsWith("flushthread=") =>
+            flushThread = threadCountStr.split("=")(1).toInt
+          case illegal =>
+            throw new IllegalArgumentException(s"Illegal attribute: $illegal")
+        }
+        if (flushThread == -1) {
+          flushThread = diskType match {
+            case HDD => HDDFlusherThreads
+            case SSD => SSDFlusherThreads
+          }
+        }
+        (dir, maxCapacity, flushThread, diskType)
+      }
+    }.getOrElse {
+      val prefix = workerBaseDirPrefix
+      val number = workerBaseDirNumber
+      (1 to number).map { i =>
+        (s"$prefix$i", defaultMaxCapacity, HDDFlusherThreads, HDD)
+      }
+    }
+  }
+
+  def partitionSplitMinimumSize: Long = {
+    getSizeAsBytes("rss.partition.split.minimum.size", "1m")
+  }
+
+  def hdfsDir: String = {
+    get(HDFS_DIR).map {
+      hdfsDir =>
+        if (!Utils.isHdfsPath(hdfsDir)) {
+          log.error(s"rss.worker.hdfs.dir configuration is wrong $hdfsDir. Disable hdfs support.")
```

Comment:
Please update the conf key. And "Disable hdfs support." => "Disable HDFS support."

Reply:
Done.